src/components/Card.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
**OPTIONAL** - As the previous reviewer suggested, please fix all disabled linters. Linters helps ensure that your code meets global standards 👍 👍

Reply:
@mckent05 This is an OPTIONAL remark the reviewer gave. This shouldn't prevent you from approving my work.